sound/soc/intel/boards/sof_pcm512x.c

```diff
@@ -380,6 +386,9 @@ static int sof_audio_probe(struct platform_device *pdev)
 
 	ssp_codec = sof_pcm512x_quirk & SOF_PCM512X_SSP_CODEC_MASK;
 
+	if (!(sof_pcm512x_quirk & SOF_PCM512X_ENABLE_DMIC))
+		dmic_be_num = 0;
```

Comment:
this means, that we understand why DMIC is currently working with UP2, right? This patch only adds a quirk to be able to disable DMICs?

Reply:
that's right.